lib/ui/profile/UserProfile.dart

```diff
@@ -0,0 +1,353 @@
+import 'package:farmsmart_flutter/ui/common/ActionSheet.dart';
+import 'package:farmsmart_flutter/ui/common/ListDivider.dart';
+import 'package:farmsmart_flutter/ui/common/roundedButton.dart';
+import 'package:farmsmart_flutter/ui/profile/UserProfileListItem.dart';
+import 'package:flutter/material.dart';
+import 'package:intl/intl.dart';
+
+class _Strings {
+  static final String activeCrops = "Active crops";
+  static final String completedCrops = "Completed";
+  static final String buttonTitle = "Switch Profile";
+}
+
+class UserProfileViewModel {
+  List<UserProfileListItemViewModel> items;
+  String username;
+  int activeCrops;
+  int completedCrops;
+  final Function switchProfileAction;
+  ImageProvider image;
+
+  UserProfileViewModel({
+    this.items,
+    this.username,
+    this.activeCrops,
+    this.completedCrops,
+    this.switchProfileAction,
+    this.image,
+  });
+}
+
+class UserProfileStyle {
+  final Color buttonColor;
+  final Color dividerColor;
+
+  final TextStyle titleTextStyle;
+  final TextStyle subtitleTextStyle;
+  final TextStyle detailTextStyle;
+  final TextStyle buttonTextStyle;
+
+  final EdgeInsets edgePadding;
+  final EdgeInsets headerEdgePadding;
+  final EdgeInsets titleLineSpace;
+
+  final BorderRadius buttonBorderShape;
+
+  final double imageSpacing;
+  final double subtitleLineSpace;
+  final double buttonLineSpace;
+  final double detailSpacing;
+  final double dividerHeight;
```

Comment:
Dividers, paddings, line spaces, divider color, etc... Should go on `_Constants` we will never change this. Just add the params that you think we want to change, for example: textstyles, button colors, max lines, image size, button size... 😄

Reply:
Done